### ISSUE TITLE
Please do not delete the publish directory  when it is published

### ISSUE THREAD

**Issue by dpull** [cocos2d/CocosBuilder#386]
I set to publish only ccbi files to the current directory，do not delete the publish directory when it is published.

![cxxcxc](https://f.cloud.github.com/assets/3412769/672520/05557e60-d887-11e2-8118-ad2d434c93b5.jpg)

**Comment by emmychen** [comment]
I had the same problem here.  The alpha - 4 is ok.  but alpha 5 delete the publish directory with all the resources file.   Please fix it.  Thanks

**Comment by bachirelkhoury** [comment]
Same problem here!!! it wiped ALL resources!! common!!
and the worse thing is the latest cocos2d-x 2.1.4 doesn't work with CocosBuilder 2.1.
I get
WARNING! Incompatible ccbi file version (file: 3 reader: 5)

**Comment by ixqbar** [comment]
Same problem here!!!

**Comment by Tw0bit** [comment]
Any sign of beta6 coming? I too have lost a shared resources folder (luckily restored from git).
Reverted to Beta4...

**Comment by huperniketes** [comment]
This behavior isn't even close to being user-friendly. Without so much as a warning, a project directory is deleted. This must be remedied.

**Comment by myeviltwin** [comment]
Yes, I have this problem too.  Can't find a way to use the latest cocos2d-iphone 2.1 "coordinated release" with cocosbuilder anymore.

**Comment by niiiico** [comment]
I've checked the code and found out why this is happening : 
Weither or not Android and HTML5 is selected in the Publish settings screen it is trying to clean the directories.
As the directory path is not entered it assumes it is the current dir and therefore delete the directory of the project.

What I'm doing now I'm generating Android and HTML5 files in the same subfolder as iOS and it is working fine.

Hope this helps.
